WcaOnRails/app/webpacker/packs/homepage.js

```diff
@@ -0,0 +1,8 @@
+import 'semantic-css/button';
+import 'semantic-css/divider';
+import 'semantic-css/header';
+import 'semantic-css/image';
+import 'semantic-css/placeholder';
+import 'semantic-css/segment';
+
+import '../stylesheets/homepage.scss';
```

Comment:
Just to make sure: if we want to use any of the semantic UI classes (except the `minimal_css` ones) on normal pages, we need to create a corresponding pack including the appropriate components and then add that pack to the given page, right?

Reply:
Yes!
Not very convenient unfortunately :(
Maybe I'm overthinking this and we should just go with all the CSS in one place, and living with the fact that the whole thing will be heavy anyway.
I'll take a look again at the packs and try to give accurate numbers.